firmware/targets/f7/furi_hal/furi_hal_speaker.c

```diff
@@ -1,23 +1,57 @@
 #include <furi_hal_speaker.h>
 #include <furi_hal_gpio.h>
 #include <furi_hal_resources.h>
+#include <furi_hal_power.h>
 
 #include <stm32wbxx_ll_tim.h>
+#include <furi_hal_cortex.h>
+
+#define TAG "FuriHalSpeaker"
 
 #define FURI_HAL_SPEAKER_TIMER TIM16
 #define FURI_HAL_SPEAKER_CHANNEL LL_TIM_CHANNEL_CH1
 #define FURI_HAL_SPEAKER_PRESCALER 500
 #define FURI_HAL_SPEAKER_MAX_VOLUME 60
 
+FuriMutex* furi_hal_speaker_bus_mutex = NULL;
+
 // #define FURI_HAL_SPEAKER_NEW_VOLUME
 
 void furi_hal_speaker_init() {
+    furi_hal_speaker_bus_mutex = furi_mutex_alloc(FuriMutexTypeNormal);
```

Comment:
assert that furi_hal_speaker_bus_mutex is NULL, or free if it's not null

Reply:
No need